File: code-examples/java/products/update_variant.java
HttpResponse<String> response = Unirest.put("http://api.marketcloud.it/v0/products/:id/variants/:variant_id")
  .header("authorization", "YourPublicKey:YourToken")
  .header("accept", "application/json")
  .header("content-type", "application/json")
  .body("{\"price\" : 20}")
  .asString();

File: code-examples/csharp/products/update_variant.cs
var client = new RestClient("http://api.marketcloud.it/v0/products/:id/variants/:variant_id");
var request = new RestRequest(Method.PUT);
request.AddHeader("content-type", "application/json");
request.AddHeader("accept", "application/json");
request.AddHeader("authorization", "YourPublicKey:YourToken");
request.AddParameter("application/json", "{\"price\" : 20}", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);